README.md

```diff
@@ -244,6 +244,40 @@ type Output struct {
 }
```

Comment:
Could you add a test where you have an exact match and matches in different cases, the result should be that the exact match is used:
```go
package structs

// goverter:converter
type Converter interface {
    // goverter:fold
    Convert(source Input) Output
}

type Input struct {
    Name string
    NAME string
}
type Output struct {
    Name string
}
```

Reply:
Already there in (now renamed) scenario/3_struct_match_ignore_case_ambiguous_ok.yml, please see OtherID and others